src/components/molecules/Visualizer/Plugin/api.ts

```diff
@@ -156,12 +160,18 @@ export function commonReearth({
         get position() {
           return camera();
         },
+        get viewport() {
+          return viewport;
+        },
       },
       get property() {
         return sceneProperty();
       },
     },
     layers: {
+      get layersInViewport() {
+        return layersInViewport;
```

Comment:
The plugin API often uses getters, so make sure to return the value directly instead of returning a function.

```suggestion
        return layersInViewport();
```